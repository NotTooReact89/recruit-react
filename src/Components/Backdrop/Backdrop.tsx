import * as React from 'react';
import { IClickProps } from '..';

import './Backdrop.css';

const Backdrop = (props: IClickProps) => (
    <div className="backdrop" onClick={props.click} />
);

export default Backdrop;