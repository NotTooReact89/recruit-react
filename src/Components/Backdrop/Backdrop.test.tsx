import * as React from 'react';
import Backdrop from './Backdrop';
const renderer = require('react-test-renderer');

it('should render backdrop correctly', () => {
  const tree = renderer.create((
    <Backdrop click={jest.fn()} />
  )).toJSON()
  expect(tree).toMatchSnapshot()
})