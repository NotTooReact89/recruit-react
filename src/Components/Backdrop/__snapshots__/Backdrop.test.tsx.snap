// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render backdrop correctly 1`] = `
<div
  className="backdrop"
  onClick={[MockFunction]}
/>
`;
